routes/web.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use App\Http\Controllers\Auth\ArticleController;
+use App\Http\Controllers\ArticleController as HomeArticleController;
+use App\Http\Controllers\Auth\CategoryController;
+use App\Http\Controllers\Auth\TagController;
+use App\Http\Controllers\HomeController;
+use Illuminate\Support\Facades\Route;
+
+Route::get('/', [HomeController::class, 'index'])->name('home');
+
+Route::get('/about-me', fn() => view('about'))->name('about');
+
+Route::get('/articles/{article}', [HomeArticleController::class, 'show'])->name('articles.show');
+
+Route::prefix('dashboard')->middleware(['auth'])->group(function () {
+
+    Route::get('/', fn () => view('dashboard'))->name('dashboard');
```

Comment:
`Route::view` can be used [docs link](https://laravel.com/docs/8.x/routing#view-routes)